nilearn/datasets/atlas.py

```diff
@@ -11,7 +11,7 @@
 
 from .utils import _get_dataset_dir, _fetch_files, _get_dataset_descr
 from .._utils import check_niimg
-from .._utils.compat import _basestring, get_affine
+from .._utils.compat import _basestring, get_affine, get_header
```

Comment:
We just bumped nibabel to 2.0.2. I think these imports get_affine and get_header can be removed.